spotbugs/etc/messages.xml

```diff
@@ -7304,6 +7312,19 @@ If the object is, indeed, non-serializable, an error will result.
 This format string includes a newline character (\n). In format strings, it is generally
  preferable to use %n, which will produce the platform-specific line separator.
 </p>
+]]>
+     </Details>
+  </BugPattern>
+<BugPattern type="FS_BAD_DATE_FORMAT_FLAG_COMBO">
+    <ShortDescription>Certain SimpleDateFormat format strings may lead to unexpected behavior</ShortDescription>
+    <LongDescription>Certain SimpleDateFormat format strings may lead to unexpected behavior in {1}</LongDescription>
+    <Details>
+<![CDATA[
+<p>
+This format stream includes a bad combination of format flags which may lead to unexpected behavior. Examples include
+using a week year ("Y") without specifying week in year ("w") or using an AM/PM hour ("h" or "K") without specifying
+an AM/PM marker ("a").
```

Comment:
Consider providing the list of full examples.
Users will not read SpotBugs source code, we need a document or URL to tell 'what is the problem' and 'how you can fix it'.